restli-tools/src/main/resources/apiVmTemplates/simple_resource.vm

```diff
@@ -51,7 +51,7 @@ import java.util.concurrent.CompletableFuture;
 import javax.annotation.Generated;
 
 @Generated("Generated from $spec.sourceIdlName")
-public class $spec.className {
+public class ${spec.className}${class_name_suffix}{
```

Comment:
This template simple_resource.vm is not used anywhere, I guess you forgot to delete this?

Reply:
thanks. deleted